packages/pancake-uikit/src/widgets/Modal/Modal.tsx

```diff
@@ -48,9 +51,11 @@ const Modal: React.FC<Props> = ({
   children,
   hideCloseButton = false,
   bodyPadding = "24px",
+  headerBackground = "transparent",
+  minWidth = "360px",
```

Comment:
Why put the minWidth on the title and not the parent?

Reply:
@hachiojidev 

minWidth is added to the parent. It just looks like that it is added to the Title in the git. :)

Reply:
My bad.